app/components/Vault/index.js

```diff
@@ -366,6 +366,21 @@ const Vault = (props) => {
     tokenBalance = ethBalance;
   }
 
+  const vaultBalanceDecimalPlacesCount = (() => {
```

Comment:
This looks good but can we update to 4 decimals for all vaults?

Reply:
done